storage_test.go

```diff
@@ -26,75 +27,47 @@ func initDb() error {
 		AwsRegion:     os.Getenv("AWS_DEFAULT_REGION"),
 		AwsDisableSSL: true,
 	}
-	sess, err := session.NewSession(&aws.Config{
-		Endpoint:   &storage.AwsEndpoint,
-		Region:     &storage.AwsRegion,
-		DisableSSL: &storage.AwsDisableSSL,
-	})
-	if err != nil {
+
+	ctx := context.Background()
+	if err := storage.initConfig(ctx); err != nil {
 		return err
 	}
 
-	svc := dynamodb.New(sess)
-
 	// attempt to delete table in case already exists
 	deleteTable := &dynamodb.DeleteTableInput{
 		TableName: aws.String(storage.Table),
 	}
-	_, err = svc.DeleteTable(deleteTable)
-	if err != nil {
-		if aerr, ok := err.(awserr.Error); ok {
-			switch aerr.Code() {
-			case dynamodb.ErrCodeResourceNotFoundException:
-				// this is fine
-			default:
-				return aerr
-			}
-		} else {
-			return err
-		}
-	}
+	_, _ = storage.Client.DeleteTable(ctx, deleteTable)
```

Comment:
is this not expected to fail?

Reply:
It may fail if the table already exists. The problem is that it's not feasible to detect only a table-exists error. I'm not too concerned since it's only for test.